app/Helpers/ShortenUrl.php

```diff
@@ -1,101 +1,76 @@
 <?php
-namespace ChopBox\helpers;
-
 
+namespace ChopBox\helpers;
 
-class ShortenUrl
-{
-
+class ShortenUrl {
   private $api_version = "v3";
   private $format;
   private $login;
   private $api_key;
 
-
-  public function setKey($api_key)
-  {
+  public function setKey($api_key) {
     $this->api_key = $api_key;
   }
 
-  public function setLogin($login)
-  {
+  public function setLogin($login) {
     $this->login = $login;
   }
 
-  public function setFormat($format)
-  {
+  public function setFormat($format) {
     $this->format = $format;
   }
 
-
-  public function shortenUrl($url)
-  {
+  public function shortenUrl($url) {
     $shortened_url = "";
     $encoded_url = urlencode($url);
-    $bityl_url = "http://api.bit.ly/shorten?" . 
-    "version=" . $this->api_version . 
-    "&format=" . $this->format . 
-    "&longUrl=" . $encoded_url . 
-    "&login=" . $this->login . 
-    "&apiKey=" . $this->api_key;
-
+    $bityl_url = "http://api.bit.ly/shorten?" . "version=" . $this->api_version . "&format=" . $this->format . "&longUrl=" . $encoded_url . "&login=" . $this->login . "&apiKey=" . $this->api_key;
+    
     $content = file_get_contents($bityl_url);
-
-    try
-    {
-      $shortened_url = $this->parseContent($content, $url);
     
-    }catch(Exception $e)
-    {
-       echo "Caught exception: " . 
-       $e->getMessage() ."<br/>";
-       exit;
+    try {
+      $shortened_url = $this->parseContent($content, $url);
+    } catch ( Exception $e ) {
+      echo "Caught exception: " . $e->getMessage() . "<br/>";
```

Comment:
Trust me, you don't want to throw this kind of stack trace to the user. Don't echo, instead return an `appropriate message. I mean that's why you are catching the Exception in the first palce.